src/booking-list/datesManager.ts

```diff
@@ -0,0 +1,38 @@
+import { amountOfDays } from 'src/booking-list/constants';
+
+class DatesManager {
+  getDatesMS() {
+    let arrOfDays: number[] = [];
+    arrOfDays.length = amountOfDays - 1;
+    arrOfDays.fill(0);
+
+    let currentDate = new Date();
```

Comment:
try to use `const` for things you don't mutate in subsequent code

Reply:
did